tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/JSGen.scala

```diff
@@ -147,27 +151,42 @@ private[emitter] final class JSGen(val semantics: Semantics,
   }
 
   def genRawJSClassConstructor(className: String)(
-      implicit globalKnowledge: GlobalKnowledge, pos: Position): Tree = {
+      implicit globalKnowledge: GlobalKnowledge,
+      pos: Position): WithGlobals[Tree] = {
```

Comment:
Oh, sorry. I just meant the one without the default value for `keepAllGlobalVarNames`. `genRawJSClassConstructor` can stay.